src/target/riscv32.c

```diff
@@ -101,6 +105,11 @@ bool riscv32_probe(target_s *const target)
 	case JEP106_MANUFACTURER_RV_GIGADEVICE:
 		PROBE(gd32vf1_probe);
 		break;
+#if 0
+	case JEP106_MANUFACTURER_WCH:
+		PROBE(ch32v3xx_probe);
+		break;
+#endif
```

Comment:
This looks to be code left over from another potential PR, please remove it from this one.